doc/pages/FAQ.md

```diff
@@ -437,3 +437,31 @@ We don't recommend using it in day-to-day use of opam in the shell, because
 you'll be typing more and you won't get to notice exciting new features! If the
 behaviour of a command or option is altered, and you write something which in no
 longer valid, opam will try to tell you what the new command should look like.
+
+---
+
+#### 🐫  How does `opam init` and `opam switch create <version>` know which compiler package to choose?
+
+Since opam 2.4, `opam init` will choose `ocaml-base-compiler` as default
+compiler for the `default` switch, and let the solver choose the latest
+version. Of course this default can always be changed using `--compiler`
+or by skipping the creation of the `default` switch all-together with
+`--bare`.
```

Comment:
```suggestion
Since opam 2.4, `opam init` will select `ocaml-base-compiler` as the default
compiler package for the `default` switch, and let the solver choose the latest
version. This can always be changed using `--compiler` or by skipping the
creation of the `default` switch altogether with `--bare`.
```